app/views/users/_sign_in_message.html.haml

```diff
@@ -0,0 +1,6 @@
+- if @sign_up
```

Comment:
Generally good practise to avoid the use of member variables in partials. Better to refactor to `sign_up` with `signup: @sign_up` used in the `render` in the controller.